### PR TITLE
ENDOC-382

### PR DESCRIPTION
**Pull request by Lyd1aCla1r3**
First pass at editing existing content

### CODE REVIEW

File: vuepress/docs/next/tutorials/ecr/ecr-private-git-repo.md [hunk 1]
```diff
@@ -4,90 +4,79 @@ redirectFrom:/next/docs/ecr/ecr-setup-private-git.html
...
 
-Entando currently allows a Kubernetes Secret containing a Git SSH private key to be mounted in the container that hosts the Entando Component Manager service. This container is deployed with the EntandoApp and can therefore be configured
-from the EntandoApp custom resource. The best approach to prepare such a secret is by generating the keypair on 
-your local machine using a Docker image, and creating a Secret from the directory the keypair was generated in.
+Entando allows a Kubernetes Secret containing a Git SSH private key to be mounted in the container hosting the Entando Component Manager service. This container is deployed with the EntandoApp and can be configured from the EntandoApp custom resource. To prepare a Secret, first generate the keypair locally using a Docker image, then create the Secret from the directory where the keypair was generated.
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/ecr/ecr-private-git-repo.md:16]: Capitalize Custom Resource?

**Review by nshaw — CHANGES_REQUESTED** [review]
I think we also need a note in the Overview about requiring the SSH style URL instead of HTTPS URL for this to work, etc.  git@github.com:THE-REPO-OWNER-NAME/THE-REPO-NAME.git instead of https://github.com/THE-REPO-OWNER-NAME/THE-REPO-NAME.git

File: vuepress/docs/next/tutorials/ecr/ecr-private-git-repo.md (hunk 1 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/ecr/ecr-private-git-repo.md:16]: 👍 i was on the fence

**Review by nshaw — APPROVED** [review]

File: vuepress/docs/next/tutorials/ecr/ecr-private-git-repo.md [hunk 2]
```diff
@@ -4,90 +4,85 @@ redirectFrom:/next/docs/ecr/ecr-setup-private-git.html
 # Install Bundles from a Private Git Repository
 
 ## Overview
-The standard deployment of the Entando Component Repository assumes that bundles are checked out from public Git repositories. Public repositories do not require user authentication in order to access content inside the Git repository.
+The standard deployment of the Entando Component Repository (ECR) assumes that bundles are checked out from public Git repositories. Public repositories do not require user authentication, but if a customer chooses to limit repository access to specific Git accounts a private key is required. This document details how to allow the ECR to access a private Git repository.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/ecr/ecr-private-git-repo.md:7]: "to specific Git accounts, a private key is required"  --comma is needed here.

File: vuepress/docs/next/tutorials/ecr/ecr-private-git-repo.md [hunk 3]
```diff
@@ -4,90 +4,85 @@ redirectFrom:/next/docs/ecr/ecr-setup-private-git.html
...
 1. [Entando custom resource reference](../../docs/concepts/custom-resources.md)
 2. [Entando Component Repository overview](../../docs/ecr/ecr-overview.md)
 
 ## Using SSH keys with Git
-Entando supports SSH keys to connect to Git repositories. Following the SSH approach, a developer that 
-connects to Git would typically generate a public/private keypair. The developer would then register the public
-key with the Git server, whilst storing the private key securely in the local filesystem. When the Git command line
-utility executes an operation that requires an authenticated user, it will automatically look for the private
-key locally and match it with the public key provided by the server. If the two match up, then the operation is allowed to complete.
+Entando supports connecting to Git repositories with SSH keys. While SSH Git URLs are accepted, HTTPS git URLs are not. For the private key to work correctly the SSH syntax of `git@github.com:THE-REPO-OWNER-NAME/THE-REPO-NAME.git` must be used (the HTTPS syntax of `https://github.com/THE-REPO-OWNER-NAME/THE-REPO-NAME.git` is not permitted).
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/ecr/ecr-private-git-repo.md:14]: "For the private key to work correctly, the SSH syntax of `git@github.com:THE-REPO-OWNER-NAME/THE-REPO-NAME.git` must be used.."

sorry another comma needed.

File: vuepress/docs/next/tutorials/ecr/ecr-private-git-repo.md [hunk 4]
```diff
@@ -4,90 +4,85 @@ redirectFrom:/next/docs/ecr/ecr-setup-private-git.html
...
+
+Using the SSH method a developer can generate a public/private keypair, then register the public key with the Git server while securely storing the private key locally. An operation requiring authentication will trigger the Git command line utility to perform a search and compare between the local private key and the public key provided by the server. If the two keys are identical the operation is allowed to complete.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/ecr/ecr-private-git-repo.md:16]: If the two keys are identical, the operation is allowed to complete. -- comma 

I am not going around looking to add commas, honestly. This is what I follow, If you want to check it out: https://owl.purdue.edu/owl/general_writing/punctuation/commas/commas_after_introductions.html

Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/ecr/ecr-private-git-repo.md:16]: what kind of introductory phrase is this??

**Review by jyunmitch — APPROVED** [review]

**Pull request merged**